lib/ui/notification/update/change_log_page.dart

```diff
@@ -105,27 +105,28 @@ class _ChangeLogPageState extends State<ChangeLogPage> {
 
     items.add(
       ChangeLogEntry(
-        "Security Audit",
-        "It gives us immense joy to announce that ente's source code has been"
-            " successfully audited by Cure53, in collaboration with Symbolic "
-            "Software.\n\nTogether they have certified that ente's "
-            "architecture is sound and that our implementation across all "
-            "clients is cryptographically accurate.\n\nYou can find more "
-            "details at ente.io/blog.",
+        "Motion photos",
+        "ente Mobile App now supports playing Motion photos. Long-press on "
```

Comment:
We now support the playback of Motion Photos. Open a Motion Photo, and long press to play the video.